python/py-crossbar/Portfile

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-crossbar
+version             20.8.1
+revision            0
+categories          python net
+platforms           darwin
+license             AGPL-3
+
+homepage            http://crossbar.io/
```

Comment:
Use `https://crossbar.io`. This http server redirects to https. And when using only a hostname, we now drop the trailing `/`, matching the behavior web browsers adopted some years ago.

Reply:
Fixed